packages/client/src/execution/vmexecution.ts

```diff
@@ -271,168 +279,170 @@ export class VMExecution extends Execution {
     if (this.running || !this.started || this.config.shutdown) return 0
     this.running = true
 
-    // run inside a lock so as to not be entangle with runWithoutSetHead or setHead
-    return this.runWithLock<number>(async () => {
```

Comment:
the diff here is because of the try catch added so that `this.running` can be turned to `false` in the `finally` section

Reply:
👍